File: java/AppendText.java
import java.util.Scanner;

public class AppendText {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first text: ");
        String first = scanner.nextLine();
        System.out.print("Enter second text: ");
        String second = scanner.nextLine();
        scanner.close();

        first = first + second;
        System.out.println("Append text: " + first);
    }
}

File: kt/AppendText.kt
import java.util.Scanner

fun main() {
    var scanner = Scanner(System.`in`)
    print("Enter first text: ")
    var first: String = scanner.nextLine()
    print("Enter second text: ")
    var second: String = scanner.nextLine()

    first = first + second
    println("Append text: " + first)
}